### PR TITLE
Remove the rest of the safeStrto* function from conversion.h

### PR DESCRIPTION
**Pull request by akindyakov**
Used `tryTo<>` from #4676 instead

### CODE REVIEW

File: osquery/core/windows/process_ops.cpp [hunk 1]
```diff
@@ -24,31 +24,32 @@ std::string psidToString(PSID sid) {
...
   auto toks = osquery::split(sidString, "-");
 
   if (toks.size() < 1) {
     LocalFree(sidString);
-    return uid;
+    return uid_default;
   }
 
-  auto ret = safeStrtoul(toks.at(toks.size() - 1), 10, uid);
+  auto uid_exp = tryTo<unsigned long int>(toks.at(toks.size() - 1), 10);
```
Comment (guliashvili) [inline osquery/core/windows/process_ops.cpp]: Why do we have unsigned long default variable and extract unsigned long when int is returned?
It was like that before, so feel free to ignore my comment.

Comment (guliashvili) [reply osquery/core/windows/process_ops.cpp]: toks.at(toks.size() - 1) -> toks.back()

again feel free to ignore.

File: osquery/core/windows/process_ops.cpp [hunk 2]
```diff
@@ -82,7 +83,7 @@ int getGidFromSid(PSID sid) {
     LPTSTR sidString;
     ConvertSidToStringSid(sid, &sidString);
     auto toks = osquery::split(sidString, "-");
-    safeStrtoul(toks.at(toks.size() - 1), 10, gid);
+    gid = tryTo<unsigned long int>(toks.at(toks.size() - 1), 10).takeOr(gid);
```
Comment (guliashvili) [inline osquery/core/windows/process_ops.cpp]: toks.at(toks.size() - 1) -> toks.back()

again feel free to ignore.

File: osquery/core/windows/process_ops.cpp (hunk 1 above)
Comment (akindyakov) [reply osquery/core/windows/process_ops.cpp]: Good catch, but I have no idea why it is implemented this way :)
I'd like to keep changes as small as I can. We can fix it in the next PR.

**Comment by akindyakov** [comment]
jenkins test linux

File: osquery/core/windows/process_ops.cpp (hunk 1 above)
Comment (guliashvili) [reply osquery/core/windows/process_ops.cpp]: Sure, will do it later maybe. Assume those are notes for me :D

**Pull request merged**